Podfile

```diff
@@ -4,9 +4,27 @@ use_frameworks!
 target 'Polls' do
   pod 'Hyperdrive'
   pod 'SVProgressHUD'
+  pod 'VTAcknowledgementsViewController'
 end
 
 target 'PollsTests' do
 
 end
 
+post_install do |installer|
+  # Inject the plist acknowledements file into the resource
+  # To be moved to CP plugin: https://github.com/CocoaPods/CocoaPods/issues/2465
+  resources = File.read('Pods/Target Support Files/Pods-Polls/Pods-Polls-resources.sh').split("\n")
+  rsync_index = resources.index { |line| line =~ /^rsync/ }
+  resources.insert(rsync_index, 'install_resource "Target Support Files/Pods-Polls/Pods-Polls-acknowledgements.plist"')
+  File.write('Pods/Target Support Files/Pods-Polls/Pods-Polls-resources.sh', resources.join("\n"))
```

Comment:
It certainly is brittle and tied to implementation details of CocoaPods, it will be replaced with a CocoaPods plugin sometime which allows this to work better: https://github.com/CocoaPods/CocoaPods/issues/2465